ore/security/ir.model.access.csv

```diff
@@ -17,6 +17,8 @@ ore_membre_access_user_types_/_internal_user,ore_membre Access User types / Inte
 ore_membre_access_user_types_/_portal,ore_membre Access User types / Portal,model_ore_membre,base.group_portal,1,1,1,1
 ore_membre_favoris_access_user_types_/_internal_user,ore_membre_favoris Access User types / Internal User,model_ore_membre_favoris,base.group_user,1,1,1,1
 ore_membre_favoris_access_user_types_/_portal,ore_membre_favoris Access User types / Portal,model_ore_membre_favoris,base.group_portal,1,1,1,1
+access_ore_membre_interet,access_ore_membre_interet,model_ore_membre_interet,base.group_user,1,1,1,1
```

Comment:
Il semble manquer aussi base.grou_portal

Reply:
Cela cause une erreur si manquant : 
Access Denied by ACLs for operation: read, uid: 6, model: ore.membre.interet
Le problème, l'utilisateur n'est pas informé de cette erreur. Il faudrait lui afficher un message d'erreur.